NumberSystem/Median.java

```diff
@@ -0,0 +1,70 @@
+package NumberSystem;
+import java.util.Arrays;
+/**
+ * This class returns the median value of an array.
+ * Median is the middle element of the array after it is
+ * sorted. If there are even elements in the array median value
+ * is the value returned after adding  the two values
+ * and then dividing by 2.
+ * <br>
+ * e.g [1,2,3] has median 3
+ * e.g [1,2,3,4] has median 2.5 (2+3) /2
+ * @author Kostas
+ *
+ * version 1.0
+ */
+public class Median {
+	
+	/*
+	 * Usage: Give as many numbers seperated by a space.
+	 */
+	public static void main(String args[]) {
+		double[] doubleValues = Arrays.stream(args)
+                .mapToDouble(Double::parseDouble)
+                .toArray();
+		System.out.println(Arrays.toString(doubleValues));
+		System.out.println(Median.getMedian(doubleValues));	
+	}
+	
+	
+	
+	/**
+	 * This method returns the median value of an array.
+	 * Note that the returned value is always boolean.
+	 * @param array an array of double values
+	 * @return the median value
+	 */
+	public static double getMedian(double[] array) {
+		Arrays.sort(array);
+		//if there are even elements
+		if(array.length % 2 == 0) {
+			//add the two in the middle and divide by 2
+			return (array[array.length/2 - 1] + array[array.length/2]) / 2; 
+		}
+		//else return the middle one
+		return array[(array.length-1)/2];
+	}
+	
+	
+	
+	/**
+	 * This method returns the median value of an array.
+	 * Note that the returned value is always boolean despite the int[] input.
```

Comment:
I think you meant the returned value is always double.

Reply:
About that :
> Run these commands to pass all checks. 

I messed up with my VM so i don't have linux at the time. Is there a way to run them at windows?
Good catch. Thanks!
>I think you meant the returned value is always double.

I am new at this this is my second PR so I do make some newbie mistakes.
Thank you for being kind and understanding!

Reply:
Yup you can run them in powershell. These commands works well there.